new-installer/targets/aws/iac/nlb/main.tf

```diff
@@ -43,7 +34,15 @@ resource "aws_security_group_rule" "common" {
   protocol          = "TCP"
   cidr_blocks       = local.ip_allow_list
   security_group_id = aws_security_group.common.id
+}
 
+resource "aws_security_group_rule" "vpro" {
```

Comment:
vPro has multiple services. It is better to call this `mps`.
Ref: https://github.com/open-edge-platform/edge-manageability-framework/blob/main/design-proposals/vpro-opendmt.md